plugins/lookup/passwordstore.py

```diff
@@ -14,6 +14,8 @@
     description:
       - Enables Ansible to retrieve, create or update passwords from the passwordstore.org pass utility.
         It also retrieves YAML style keys stored as multilines in the passwordfile.
+      - To avoid problems when accessing multiple secrets at once, add C(auto-expand-secmem) to
+        C(~/.gnupg/gpg-agent.conf). Where this is not possible, consider using C(lock=readwrite) instead.
```

Comment:
```suggestion
        C(~/.gnupg/gpg-agent.conf). Where this is not possible, consider using I(lock=readwrite) instead.
```

Reply:
What is the logic of using I instead of C in this case?

Reply:
Ok, found the documentation... https://docs.ansible.com/ansible/2.5/dev_guide/developing_modules_documenting.html#formatting-functions

Reply:
A more up-to-date version is https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation